soc/realtek/ec/rts5912/reg/reg_rtmr.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2023 Realtek, SIBG-SD7
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * @brief RTOS Timer Controller (RTMR)
+ */
+
+typedef struct {
+	volatile uint32_t LDCNT;
+	volatile uint32_t CNT;
+
+	union {
+		volatile uint32_t CTRL;
+
+		struct {
+			volatile uint32_t EN: 1;
+			volatile uint32_t MDSEL: 1;
+			volatile uint32_t INTEN: 1;
+			uint32_t reserved1: 29;
+		} CTRL_b;
+	};
+
+	union {
+		volatile uint32_t INTSTS;
+
+		struct {
+			volatile uint32_t STS: 1;
+			uint32_t reserved1: 31;
+		} INTSTS_b;
+	};
+} RTOSTMR_Type;
+/* LDCNT */
+/* CNT */
```

Comment:
stale?

Reply:
Because the registers of the LDCNT and the CNT are only used by these field.
There are no mask field needed on it, so we will removed it in latest commit.